src/app/public/modules/datepicker/fuzzy-date.service.ts

```diff
@@ -109,21 +109,21 @@ export class SkyFuzzyDateService implements OnDestroy {
           case 'y':
             if (fuzzyDate.year) {
               dateParts.push(
-                fuzzyDateMoment.format(token)
+                fuzzyDateMoment.format(token.toUpperCase())
```

Comment:
Why are we converting the tokens to upper case?

Reply:
Disregard... looking at moment.js docs, it looks like the lower case tokens were wrong! Good catch.

Reply:
So I actually think I may revert this and file a new bug. We aren't really formatting correctly here. I would like to look into that but I don't want to muddy this PR. I was attempting to fix an issue where lower case in the format string causes a bad result; however, this fix doesn't really go all the way as you could want something with the day of the week and this function doesn't allow that in general. So I think a separate bug to address that is more appropriate than this Band-Aid.

Reply:
I've reverted for now and fixed the tests that were failing by just using the correct format strings.